oaknational/Oak-Web-Application
# Feat: webinar listing page

## Description


- Webinar listing page (pageData from CMS - apart from image)
- Add video to webinar schema
- Refactor Breadcrumbs for webinar and blogs
- Refactor categories for webinar and blogs
- adds BlogsWebinarListAndCategories block component for webinars and blog index pages to reduce duplication
- update tests


## Issue(s)

Progresses #346 #348

## How to test

1. Go to {[cloud link}](https://deploy-preview-926--oak-web-application.netlify.thenational.academy/webinars)
2. Check blogs are also working as expected.

## Screenshots

How it used to look (delete if n/a):
{screenshots}

How it should now look:
{screenshots}

## Checklist
